lib/ord.js

```diff
@@ -100,15 +97,14 @@ const _getPackages = (policyLevel, appConfig) =>
 
 const _getAPIResources = (csn, appConfig, packageIds) => {
     return appConfig.serviceNames
-        .flatMap((serviceName) => createAPIResourceTemplate(serviceName, csn.definitions[serviceName], appConfig, packageIds) || [])
-        .filter((resource) => !!resource);
+        .flatMap((serviceName) => createAPIResourceTemplate(serviceName, csn.definitions[serviceName], appConfig, packageIds))
```

Comment:
@zongqichen : Function `createAPIResourceTemplate` anyways returns an empty array if there are no(public) service-resources.

Reply:
I'm thinking we should move all private functions like `_getApiResources` out of ord.js. Now we don't have unittest to protect them.

Reply:
I agree. We should do that in a separate PR.